community/ibm-cadvisor-dev/README.md

```diff
@@ -0,0 +1,89 @@
+# CAdvisor
+
+[CAdvisor](https://github.com/google/cadvisor)Analyzes resource usage and performance characteristics of running containers.
+
+```console
+$ helm install stable/ibm-cadvisor-dev
```

Comment:
Multiple references in this file to `stable/ibm-cadvisor-dev`, but this chart will be in the `community` repo.